lib/core/utils/respondable/message-id.js

```diff
@@ -0,0 +1,22 @@
+import { v1 as createUuid } from '../uuid';
+
+const messageIds = [];
+
+export function createMessageId() {
+  const uuid = createUuid();
```

Comment:
Ideally you want this to be hard to guess, in addition to being unique; v1 UUIDs only attempt to guarantee the latter and there isn't that much entropy between different UUIDs from the same source. We recommend combining a uuid with a nonce at least 32 bytes long that is generated using a cryptographically secure rng source (eg, [`window.crypto.getRandomValues`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues); note https://github.com/jsdom/jsdom/issues/1612 though).

Reply:
(more important on channel ID creation than message ID creation but good to be consistent)